src/quom/quom.py

```diff
@@ -110,6 +113,12 @@ def __write_token(self, token: Token, is_main_header: bool):
         self.__dst.write(str(token.raw))
         self.__prev_token = token
 
+    def __has_guard(self, tokens: list[Token]):
```

Comment:
To support older python versions:

```suggestion
    def __has_guard(self, tokens: List[Token]):
```